libbeat/_meta/config.reference.yml

```diff
@@ -188,7 +188,10 @@ output.elasticsearch:
   # Configure http request timeout before failing an request to Elasticsearch.
   #timeout: 90
 
-  # Use SSL settings for HTTPS.
+  # The time interval (in seconds) when metrics are set to Elasticsearch.
+  #period: 60 * time.Second
+
+  # Use SSL settings for HTTPS. Default is true.
```

Comment:
where is this doc comming from? Should be `Use SSL settings for HTTPS`.

Reply:
@urso, I'm not sure I understand your feedback, so I'm going to omit the config.reference.yml changes from this documentation-specific PR.  They can be implemented in a separate PR.

Reply:
Uhm... I would like you to modify config.reference.yml.

So originally this line said: `# Use SSL settings for HTTPS. Default is true.`
As this comment is not correct we changed it to: `# Use SSL settings for HTTPS.`

Somehow your PR undid the change back to the incorrect doc comment.

Feel free to ignore config.reference.yml for now. I can prepare another PR doing the config.reference.yml update. Btw. one also needs to run `make update` after updateing config.reference.yml, so to update all global config files we ship.